bin/linters.sh

```diff
@@ -75,6 +75,14 @@ $gometalinter \
     --vendor\
     ./...
 
+echo Checking gazelle...
+bin/gazelle
```

Comment:
This is  does not work the common use case.

1. Update a source file; add a new import
2. Run gazelle, this will update the BUILD
3. check_gazelle will fail because I have not committed the changes yet.

The correct solution is to check before and after image of the file wrt gazelle processing.
Perhaps gazelle supports such an option, if not we will have to do the comparison.